test/study-webidl.js

```diff
@@ -179,11 +207,73 @@ interface Carlos {
   attribute EventHandler onbigbisous;
 };
 `);
-    assertNbAnomalies(report, 1);
+    assertNbAnomalies(report, 2);
     assertAnomaly(report, 0, {
       name: 'unexpectedEventHandler',
       message: 'The interface `Carlos` defines an event handler `onbigbisous` but does not inherit from `EventTarget`'
     });
+    assertAnomaly(report, 1, { name: 'noEvent' });
+  });
+
+  it('follows the inheritance chain to assess event targets', () => {
+    const crawlResult = toCrawlResult(`
+[Exposed=*]
```

Comment:
maybe factor our the repeated based IDL infrastructure for events that appears in 3 tests now?
```
[Exposed=*] interface Event {};
[LegacyTreatNonObjectAsNull]
callback EventHandlerNonNull = any (Event event);
typedef EventHandlerNonNull? EventHandler;
[Exposed=*]
interface EventTarget {};
```